rpc/rpc-wrappers.go

```diff
@@ -1147,7 +1147,7 @@ func NewStorageAuthorityServer(rpc Server, impl core.StorageAuthority) error {
 	})
 
 	rpc.Handle(MethodAddSCTReceipt, func(req []byte) (response []byte, err error) {
-		var sct core.RPCSignedCertificateTimestamp
+		var sct core.SignedCertificateTimestamp
```

Comment:
Yup it should, `RPCSignedCertificateTimestamp` was just a wrapper around `SignedCertificateTimestamp` that was used to avoid the custom JSON unmarshaling function. If a old `RPCSignedCertificateTimestamp` is passed through it should have no trouble being unmarshaled into the new `core.SignedCertificateTimestamp` since the JSON unmarshaling function is no longer present.